deploy-manage/maintenance.md

```diff
@@ -1,6 +1,13 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-manage-kibana.html
+applies_to:
+  stack:
```

Comment:
I think we can leave the `stack` label out of here because all of these pages are very deployment type specific. same for all pages

```suggestion
```